app/public/js/collection/playlistsCtrl.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+app.controller('CollectionPlaylistsCtrl', function (
+    $scope,
+    $rootScope,
+    SC2apiService,
+    queueService,
+    playerService
+) {
+
+    $scope.title = 'Collection - Playlists';
+    $scope.data = '';
+    $scope.busy = false;
+
+    var clearWatch = $rootScope.$watch('userId', function (newValue, oldValue) {
+        if (newValue) {
+            clearWatch();
+            activate();
+        }
+    });
```

Comment:
Request to API requires current user id, but when controller initializes it is still undefined. Here I am waiting for a first truthy value and then activate the controller.